zwave_js/CHANGELOG.md

```diff
@@ -7,6 +7,24 @@
 - [Bump Z-Wave JS to 10.18.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v10.18.0)
 - [Bump Z-Wave JS to 10.17.1](https://github.com/zwave-js/node-zwave-js/releases/tag/v10.17.1)
 
+### Notable changes

```

Comment:
Maybe the changes relevant for users should be shown first, and the version bumps at the end of the changelog? Otherwise the interesting part vanishes below the fold.